File: backend/mongddang/src/main/java/com/onetwo/mongddang/domain/vital/dto/BloodGlucoseResponse.java
package com.onetwo.mongddang.domain.vital.dto;

import com.onetwo.mongddang.domain.vital.model.BloodGlucose;
import com.onetwo.mongddang.domain.vital.model.Vital;
import jakarta.persistence.EnumType;
import jakarta.persistence.Enumerated;
import jakarta.validation.constraints.Null;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Builder
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class BloodGlucoseResponse {
    private Long id;

    private Integer bloodSugarLevel;

    private LocalDateTime measurementTime;

    private Null content;

    @Enumerated(EnumType.STRING)
    private  Vital.GlucoseStatusType status;

    private Boolean notification;

    private String packageName;
}

File: frontend/android/app/src/main/java/com/mongddang/app/data/local/entity/BloodGlucoseResponse.kt
package com.mongddang.app.data.local.entity

import kotlinx.serialization.SerialName
import java.time.LocalDateTime

data class BloodGlucoseResponse(
    @SerialName("id")
    val id: Long?= 0L,
    @SerialName("bloodSugarLevel")
    val bloodSugarLevel: Int = 0,
    @SerialName("measurementTime")
    val measurementTime: LocalDateTime? = null,
    @SerialName("status")
    val status: Status? = null,
    @SerialName("notification")
    val notification: Boolean? = false
)
